doc/diagrams/c4-model/model.dsl

```diff
@@ -1,16 +1,15 @@
-# TODO all diagrams should be revised to resemble Energy Origin as-is
-
 apiGateway = container "API Gateway" {
     description "Routes requests to services and forwards authentication requests"
     technology "Traefik"
 }
 
-dataSyncDomain = group "Data Sync" {
-    dataSyncApi = container "Data Sync" {
-        description "Facade for DataHub 2.0"
+dataHubFacadeDomain = group "DataHubFacade Domain" {
```

Comment:
Should this also include a rename of the github repo?

Reply:
Possibly, but let us do that if and when we rename the projects in the repo.